Azure/connectors/trigger_azure_eventhub.py

```diff
@@ -139,12 +139,12 @@ async def handle_exception(self, partition_context: PartitionContext, exception:
     def run(self) -> None:  # pragma: no cover
         self.log(message="Azure EventHub Trigger has started", level="info")
 
+        loop = asyncio.new_event_loop()
+        loop.add_signal_handler(signal.SIGINT, lambda: asyncio.create_task(self.stop()))
+        loop.add_signal_handler(signal.SIGTERM, lambda: asyncio.create_task(self.stop()))
+
```

Comment:
For async connector, we use [this double while](https://github.com/SEKOIA-IO/automation-library/blob/38c33968d79da4fff4a95c5ff6a1bc08764ccc23/Salesforce/salesforce/connector.py#L148-L154) to always have an asyncio loop operational.
So, I fear that putting the loop outside the `while` will lead to another issue.